torchbenchmark/models/maskrcnn_benchmark/maskrcnn_benchmark/data/build.py

```diff
@@ -127,7 +127,7 @@ def make_data_loader(cfg, is_train=True, is_distributed=False, start_iter=0, is_
         num_iters = None
         start_iter = 0
 
-    if images_per_gpu > 1:
+    if images_per_gpu > 2:
```

Comment:
is this just a hacky way of disabling the print below? maybe just comment it all out?

Reply:
yeah, I can just delete it